src/tests/unit/platform/test_admin_grants.py

```diff
@@ -0,0 +1,280 @@
+"""Tests for the Grant and Profile objects of the CBC SDK"""
```

Comment:
This file should be renamed `test_grants.py` to match the other file rename

Reply:
Doing this and leaving my personal feelings out of it.